tests/pageobjects/assistify.page.js

```diff
@@ -40,8 +40,12 @@ class Assistify extends Page {
 		return browser.element('.external-search-content .smarti-widget .search-results');
 	}
 
-	get knowledgebasePostBtn() {
-		return browser.element('.external-search-content .smarti-widget .search-results .result-actions');
+	get knowledgebasePickAnswer() {
```

Comment:
make sure you don't mess up other tests which use this functions

Reply:
Basically, this has come from the redlink repo. So I just left it as is.